test/plugins/test_lyrics.py

```diff
@@ -14,704 +14,401 @@
 
 """Tests for the 'lyrics' plugin."""
 
-import itertools
 import os
-import re
-import unittest
-from unittest.mock import MagicMock, patch
+from functools import partial
 
-import confuse
 import pytest
-import requests
 
-from beets import logging
 from beets.library import Item
-from beets.test import _common
-from beets.util import bytestring_path
+from beets.test.helper import PluginMixin
 from beetsplug import lyrics
 
-log = logging.getLogger("beets.test_lyrics")
-raw_backend = lyrics.Backend({}, log)
-google = lyrics.Google(MagicMock(), log)
-genius = lyrics.Genius(MagicMock(), log)
-tekstowo = lyrics.Tekstowo(MagicMock(), log)
-lrclib = lyrics.LRCLib(MagicMock(), log)
+PHRASE_BY_TITLE = {
+    "Lady Madonna": "friday night arrives without a suitcase",
+    "Jazz'n'blues": "as i check my balance i kiss the screen",
+    "Beets song": "via plugins, beets becomes a panacea",
+}
 
+_p = pytest.param
```

Comment:
I suggest removing this alias, it's not necessary and it makes things confusing imo.

Reply:
Agree that there's no point for it anymore!

I prefer this alias over `pytest.param` because it's less verbose, though this only matters in cases such as below, where each test case is in a single line:

```python
    @pytest.mark.parametrize(
        "drop, keep_in_artist, expected_artist, expected_title",
        [
            _p(False, False, ARTIST, TITLE_WITH_FT, id="move-to-title"),
            _p(False, True, ARTIST_WITH_FT, TITLE_WITH_FT, id="copy-to-title"),
            _p(True, False, ARTIST, TITLE, id="drop"),
            _p(True, True, ARTIST_WITH_FT, TITLE, id="keep-in-place"),
        ],
    )
    def test_handle_ft(
```